cypress/integration/e2e/homePage.js

```diff
@@ -0,0 +1,54 @@
+//import { LOCAL } from "../helper/constants"
+
+context( 'Test homepage', () => {
+    before( () => {
+        cy.visit( 'http://stage.one.redhat.com/' ); Cypress.config( 'includeShadowDom', true ); cy.get( '#username', { timeout: 5000 } ).type( 'one-portal-test' ); cy.get( '#password' ).type( 'iequeivaequ:e2OobahM3vahbaePh8' ); cy.get( '#submit' ).click();
```

Comment:
```suggestion
        cy.visit( 'http://stage.one.redhat.com/' ); Cypress.config( 'includeShadowDom', true ); cy.get( '#username', { timeout: 5000 } ).type( 'one-portal-test' ); cy.get( '#password' ).type( 'iequeivaequ:e2OobahM3vahbaePh8' ); cy.get( '#submit' ).click();
```

Test username and passwords are exposed its a security breach. I request to please change this